cirq/iqm/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from cirq.iqm.iqm_device import Adonis
```

Comment:
Can you add more information here as to what the iqm package is for and what it contains?

Reply:
Will do

Reply:
Will do

Reply:
See https://github.com/quantumlib/Cirq/pull/2701/commits/c98e112b6ae252d6f167706231e6c37d41dedb2b

Reply:
@dstrain115, please see the current state, is this okay for you?

Reply:
...and https://github.com/quantumlib/Cirq/pull/2701/commits/b3e9b0272c4be4182001aeca95367b42839c39f6

Reply:
@dstrain115, please see the current state of `__init__.py` – would this be sufficient?